tests/ocpp-server/server.py

```diff
@@ -127,9 +198,16 @@ async def on_connect(websocket, path):
 
     charge_point_id = path.strip("/")
     cp = ChargePoint(charge_point_id, websocket)
+
     ChargePoints.add(cp)
     try:
         await cp.start()
+        # Check if request argument is specified
+        if args.request:
```

Comment:
cf. previous comment. Not at connection successful opening. It must by designed to handle any change after the OCPP2 server startup.